tests/test_person.py

```diff
@@ -1,57 +1,57 @@
 import datetime
 
-from django.core.urlresolvers import reverse
+from django.urls import reverse, resolve
 import pytest
 
 from .conftest import get_uid_chunk
-from lametro.views import LAMetroPerson
+from lametro.views import PersonDetailView, LAPersonDetailView
 
 
 @pytest.mark.django_db
-def test_person_page_redirects(client, metro_person, mocker):
+def test_person_page_redirects(client, metro_person, mocker, rf):
     person = metro_person.build()
 
-    # Patch a bunch of model attributes, because patching get_context_data
-    # breaks the url routing, for some reason.
+    response = mocker.Mock()
+    response.status_code = 200
 
-    mock_membership = mocker.patch('lametro.views.Membership', autospec=True)
-    mock_membership.role = 'Superhero'
-
-    mocker.patch.object(LAMetroPerson, 'latest_council_membership', return_value=mock_membership)
-
-    mocker.patch.object(LAMetroPerson, 'current_council_seat', return_value=None)
-    mocker.patch.object(LAMetroPerson, 'committee_sponsorships', return_value=[])
+    mocker.patch.object(PersonDetailView, 'dispatch', return_value=response)
```

Comment:
Why do you need to do this?

Reply:
this test is all about correct routing, but we don't actually need to return the objects that come back from successfully dispatched routing. 

doing it this way reduced the amount of mocking I needed to do.

Reply:
Cool, could you leave a comment inline? Looking at this the first time made me go "Hmmmm"

Reply:
done.